src/remark/controllers/inputs/mouse.js

```diff
@@ -6,6 +6,17 @@ exports.unregister = function (events) {
   removeMouseEventListeners(events);
 };
 
+function throttle(events, eventName) {
+  var timestamp = null;
+  return function() {
+    var now = Date.now();
+    if (timestamp === null || now - timestamp > 100) {
```

Comment:
What do you think about making this optional?

Right now you can enable/disable scrolling - would it make sense to add an option to throttle and the time value e.g.:

```
var slideshow = remark.create({
  navigation: {
      scrollThrottle: 100,  // default value would be 0 or the same behavior we have today.
  }
});
```